src/js/components/List/List.js

```diff
@@ -220,22 +222,23 @@ const List = React.forwardRef(
                 }
               : undefined
           }
+          onTab={() => setActive(undefined)}
```

Comment:
Why do we need the `onTab` handler? I would expect `onBlur` to be sufficient.

Reply:
From what I gathered, since `onBlur` is only part of the `clickProps` events, `onBlur` and `onFocus` never get triggered when doing keyboard navigation which is why I decided to use the `onTab` prop available for Keyboard as a way to remove the active indicator upon losing focus of the List or DataTable.